api/src/endpoints/svgThumbnail.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import fs from 'fs';
+import getSvgThumbnail from 'utils/2d-map';
+
+const routes = express.Router();
+const sharp = require('sharp');
+let HPA_JSON = null;
```

Comment:
I'm not spotting where the `HPA_JSON` variable is used.

Reply:
It has been removed with the commit b6399bb7